mitiq/zne/tests/test_zne.py

```diff
@@ -31,6 +31,7 @@
     fold_gates_at_random,
     insert_id_layers,
 )
+from mitiq.zne.scaling.layer_scaling import get_layer_folding
```

Comment:
If we add `get_layer_folding` to the `scaling/__init__.py` file, we can add this import to the group directly above.

Reply:
Good call. Changed!